target/linux/ramips/image/rt305x.mk

```diff
@@ -1192,6 +1192,14 @@ define Device/zyxel_keenetic
 endef
 TARGET_DEVICES += zyxel_keenetic
 
+define Device/zyxel_keenetic-lite_rev.b
```

Comment:
Same as above, I'd go for zyxel_keenetic-lite-b here.

Reply:
renamed